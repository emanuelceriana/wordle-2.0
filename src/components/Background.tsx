import styles from "./Background.module.scss";

export const Background = () => {
  return (
    <div>
      <ul className={styles.circles}>
        <li></li>
        <li></li>
        <li></li>
        <li></li>
        <li></li>
        <li></li>
        <li></li>
        <li></li>
        <li></li>
        <li></li>
      </ul>
    </div>
  );
};
