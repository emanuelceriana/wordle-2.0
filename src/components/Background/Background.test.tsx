import { render } from "@testing-library/react";
import { Background } from "./Background";

describe("<Background />", () => {
  it("should render without errors", () => {
    const { asFragment } = render(<Background />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
