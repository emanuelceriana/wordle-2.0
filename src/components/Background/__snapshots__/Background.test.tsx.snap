// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Background /> should render without errors 1`] = `
<DocumentFragment>
  <div>
    <ul
      class="circles"
    >
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
      <li />
    </ul>
  </div>
</DocumentFragment>
`;
